#goal definition
geometry_msgs/PoseStamped[] goals
int8[] taskIds
string taskType
---
#result definition
bool isCompleted
int8[] taskIds
int8 robotId
---
#feedback
int8 robotId
int8 doorId 
time measureTime
uint8 doorStatus
